src/nft/components/profile/list/ListingButton.tsx

```diff
@@ -71,7 +72,11 @@ export const ListingButton = ({ onClick }: { onClick: () => void }) => {
             !listing.overrideFloorPrice
           )
             listingsBelowFloor.push([asset, listing])
-          else if (asset.floor_sell_order_price && listing.price >= asset.floor_sell_order_price)
+          else if (
```

Comment:
i feel like this whole code block this is in should be moved to a function because its starting to get cluttered

Reply:
good call out, I moved the issue finding logic to its own helper fn